bgpd/bgp_evpn_private.h

```diff
@@ -161,6 +161,24 @@ struct vrf_irt_node {
 #define RT_TYPE_EXPORT 2
 #define RT_TYPE_BOTH   3
 
+#define EVPN_DAD_DEFAULT_TIME 180 /* secs */
+#define EVPN_DAD_DEFAULT_MAX_MOVES 5 /* default from RFC 7432 */
+#define EVPN_DAD_DEFAULT_AUTO_RECOVERY_TIME 1800 /* secs */
+
+struct bgp_evpn_info {
```

Comment:
bgp_evpn_info name is too generic. All the information in this structure is about duplicate MAC detection. Consider changing the name of this structure.

Reply:
This structure will be used to store all evpn information, this is a starting point. Eventually moving other parameters in struct bgp which will be in separate PR.

Reply:
I think evpn related info to be kept under same struct evpn_info, lets not change around name for interim as subsequent patch of consolidation will move evpn fields under evpn_info struct. In my opinion, nested struct for dupilcate detection not required.